recipe/migrations/cuda110.yaml

```diff
@@ -61,8 +62,7 @@ cdt_name:  # [linux]
 
   - cos6   # [linux64 and os.environ.get("DEFAULT_LINUX_VERSION", "cos6") == "cos6"]
   - cos7   # [linux64 and os.environ.get("DEFAULT_LINUX_VERSION", "cos6") == "cos7"]
-  - cos6   # [linux64 and os.environ.get("DEFAULT_LINUX_VERSION", "cos6") == "cos6"]
-  - cos7   # [linux64 and os.environ.get("DEFAULT_LINUX_VERSION", "cos6") == "cos7"]
+  - cos7   # [linux64]
```

Comment:
Only this change is needed. Please remove other changes.

Reply:
Do we need the `use_local` above?

Reply:
Nope. `use_local: true` makes conda-smithy use the migrator from the feedstock which we do not want.

Reply:
Got it. Thanks for clarifying :)

Reply:
Ok should be fixed now. Please recheck